examples/scenarioLambertSolver.py

```diff
@@ -0,0 +1,576 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+
+r"""
+Overview
+--------
+
+This scenario demonstrates how to use the Lambert solver module package, consisting of
+``lambertPlanner()``, ``lambertSolver()`` and ``lambertValidator()``, the computation of a Delta-V maneuver that takes
+the spacecraft to a desired locataion in a given time.
+
+In this scenario, the goal is to reach a target position at final time tf by performing a maneuver at time tm.
+This is done by solving Lambert's problem. The Lambert problem is set up using :ref:`lambertPlanner`, which provides the
+information in the form of :ref:`lambertProblemMsgPayload` to :ref:`lambertSolver`. Lambert's problem is solved within
+:ref:`lambertSolver`, which writes the :ref:`lambertSolutionMsgPayload` and :ref:`lambertPerformanceMsgPayload` output
+messages. Finally, :ref:`lambertValidator` processes the content of those messages, computes the required Delta-V, and
+only writes a non-zero Delta-V message within :ref:`dvBurnCmdMsgPayload` if no constraints are violated
+(minimum orbit radius and final distance from targeted location) and the Delta-V solution has converged.
+
+The simulation layout is shown in the following illustration. The simulation and flight software (FSW) are divided into
+two different processes. After the maneuver, all tasks of the FSW process are disabled.
+
+.. image:: /_images/static/test_scenarioLambertSolver.svg
+   :align: center
+
+The true and measured spacecraft position and velocity are shown in the plots below.
+
+.. image:: /_images/Scenarios/scenarioLambertSolver1.svg
```

Comment:
None of the simulation images are showing up in the documentation?  Please make sure they are saved off properly, and imported with correct name and path?

Reply:
Done.